tests/test_compiler.py

```diff
@@ -927,7 +927,7 @@ def ansatz():
         results0 = ref_func(*params)
         results1 = func(*params)
         if measure_f == qml.counts:
-            ndim = 2  # both [0] and m0 are on one wire only
+            ndim = 1
```

Comment:
Updating this to the correct value `ndim=1` since these tests are single wire measurements. That way the states are properly mapped int(0)->`0` and not `00`, etc.